tests/wdio/test-harness/index.html

```diff
@@ -7,7 +7,7 @@
     <link rel="shortcut icon" href="<%= htmlWebpackPlugin.files.favicon%>">
     <title>"Terra Application"</title>
   </head>
-  <body>
+  <body tabindex="-1">
```

Comment:
I set focus to the body in some of the wdio tests to limit the presentation of the focus outline in a workflow.